static/src/javascripts/projects/common/views/experiments/measure-understanding.html

```diff
@@ -0,0 +1,12 @@
+<div class="understanding">
+    <div class="understanding__text">Do you feel better informed as a result of reading this piece?</div>
+    <button class="understanding__choice js-button" data-value="yes">
+        <%= thumbsUp %>
+        <span>Yes</span>
+    </button>
+    <button class="understanding__choice js-button" data-value="no">
+        <%= thumbsDown %>
+        <span>No</span>
+    </button>
+    <div class="understanding__feedback" hidden>Thank you!</div>
```

Comment:
Yes, but not IE10 and 09, which we still support I've heard :/ http://caniuse.com/#search=hidden